voidcrew/mapping/shuttles/box.dm

```diff
@@ -27,3 +27,31 @@
 			slots = 3,
 		),
 	)
+
+/obj/docking_port/mobile/voidcrew/box
+	name = "Box-Class Hospital Ship"
+	area_type = /area/shuttle/voidcrew/box
+	callTime = 25 SECONDS
+	dir = 2
+	port_direction = 8
+	preferred_direction = 4
+
+/area/shuttle/voidcrew/box/engine
```

Comment:
I am kinda worried about this, @SuperNovaa41 would know more than me but this wasn't needed on old voidcrew so don't know if there's alternatives than re-defining these for each ship (and would this make ships that use the same template share an area?)